content/core-features/authentication.textile

```diff
@@ -43,19 +45,19 @@ h3. Process used by client libraries connecting with basic auth:
   <img src="/images/diagrams/Ably-API-Auth1.gif" style="width: 100%" alt="Basic authentication process diagram">
 </a>
 
-Whilst Basic Authentication is simple, we recommend it to be only used on the server-side as it suffers from a number of problems:
+While basic authentication is simple, Ably recommends that it is only used server-side because of the following potential issues:
 
 * the secret is passed directly by the client to Ably, so it is not permitted for connections that are not over TLS (HTTPS or non-encrypted realtime connections) to prevent the key secret being intercepted
 * all of the configured capabilities of the key are implicitly possible in any request, and clients that legitimately obtain this key may then abuse the rights for that key
-* clients are permitted to use any client ID in all operations with Ably. As such, a client ID in messages and presence cannot be trusted as any client using Basic Authentication can masquerade with any client ID
+* clients are permitted to use any client ID in all operations with Ably. As such, a client ID in messages and presence cannot be trusted as any client using basic authentication can masquerade with any client ID
```

Comment:
I wonder if we can simplify this and also remove the word 'masquerade' for non-native English speakers?

Perhaps: "A client that authenticates using an API key can claim any client ID it chooses. Therefore this client ID  cannot be trusted to be genuine."

Reply:
Agreed, would be nice to remove that.

Reply:
Personally I think it changes the meaning. The clientId itself can't not be genuine. It should be a statement about the validity of the identity claim being made. Maybe we just say something like "Therefore this [claimed|asserted] clientId cannot be trusted to be [genuine|valid]"

Reply:
I see your point. Another suggestion for your consideration: "A client that authenticates using an API key can claim any client ID it chooses. Therefore this client ID cannot be trusted to represent the genuine identity of the client."

Reply:
I think the latter encapsulates things nicely.